nobuco/node_converters/tensor_getattr.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+import tensorflow as tf
+
+from nobuco.commons import ChannelOrderingStrategy
+from nobuco.converters.node_converter import converter
+
+
+@converter(torch.Tensor.__getattribute__, channel_ordering_strategy=ChannelOrderingStrategy.FORCE_PYTORCH_ORDER)
+def converter_getattr(self: Tensor, name: str, *, out: Optional[Tensor] = None):
+    def func(self, name: str, *, out: Optional[Tensor] = None):
+        if name == "real":
+            return tf.math.real(self)
+        elif name == "imag":
+            return tf.math.imag(self)
+        else:
+            raise AttributeError(f"'Tensor' object has no attribute '{name}'")
```

Comment:
I disagree with the error message. It's not that the original tensor does not have a certain attribute, it's that the converter doesn't support it yet. I'd suggest you to stick with the original message.

Reply:
You are right, and since the converter is needed to be modified according to your another comment, I will revisit this issue after the modification.